rental.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './person'
+class Rental
+  attr_reader :person, :book
```

Comment:
- [ ] Setters for `@book` and `@person` instance variables should be present in the Rental class.